modules/operations/pages/monitoring/stream-audit-logs.adoc

```diff
@@ -0,0 +1,39 @@
+= Astra Audit Logging
+
+Stream audit logs through {product_name} to an external system.
+
+This feature requires coordination with your DataStax representative.
+
+To enable audit logging, you will need to provide the **Full Name** of your {product_name} topic and the streaming tenant's `client.conf` file to DataStax.
+
+== Create an {product_name} topic for audit logging
+
+If you already have an existing {product_name} topic you want to use for audit logging, skip to <<download-client-conf>>.
+
+. Log in to the Astra console using your credentials.
+. Select the appropriate organization for the streaming account you wish to use.
+. Click on **Streaming** from the left-hand menu.
+. Select an existing tenant or xref:astra-streaming:getting-started:index.adoc[Create a Streaming Tenant].
```

Comment:
```
. Select an existing tenant or xref:astra-streaming:getting-started:index.adoc#your-first-streaming-tenant[Create a Streaming Tenant] in AWS `us-east-2`.
. Add a xref:astra-streaming:getting-started:index.adoc#add-a-namespace-to-hold-topics[namespace] and xref:astra-streaming:getting-started:index.adoc#a-topic-to-roganize-messages[topic] to the tenant.
. On the *Namespace and Topics* page, click the new topic, and then copy the topic's **Full Name**, such as `persistent://aws-us-east-2-mk/*NAMESPACE_NAME*/*TOPIC_NAME*.
. If necessary, create additional audit log topics, and then record the **Full Name* for each topic.
```

You can combine or delete most of these steps.

The only unusual thing here is that the tenant must be in AWS us-east-2. Otherwise, you can save a lot of space by just linking to what you've already written.

I think it would be helpful to add why you would create multiple topics.